### DISCUSSION TITLE
Feature Request: Visually distinguish internal and external links

### DISCUSSION THREAD

**Discussion by rzelnik** [zim-desktop-wiki/zim-desktop-wiki#2079] [Ideas]
I would like to have external links visually distinguished from internal links. For example Wikipedia distinguishes external links by adding this sign: ![](https://en.wikipedia.org/w/skins/Vector/resources/common/images/external-link-ltr-icon.svg?48e54) to the end of the link.

**Comment by introt** [comment]
There's an open ticket for this: #197
